### ISSUE TITLE
Better style for categories/tags page

### ISSUE THREAD

**Issue by antiagainst** [wangchucheng/hugo-eureka#125]
**Is your feature request related to a problem? Please describe.**

Right now the categories/tags page just lists all categories/tags as a plain list ([exa](https://www.lei.chat/categories)mp[les](https://www.lei.chat/tags/)), with one category/tag on each line. This leaves an annoying amount of whitespace on the categories/tags page. A better style would be preferrable.

**Describe the solution you'd like**

A more condensed style would be nice. Maybe something like [this](https://hugo-theme-even.netlify.app/categories/) and [this](https://hugo-theme-even.netlify.app/tags/)?

**Comment by wangchucheng** [comment]
I agree that whitespace will be a little more when having many terms. I will test this centered style and other styles are also welcome!
